<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>524</width>
    <height>413</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>158</width>
      <height>141</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="3" column="1">
      <widget class="QLabel" name="OutValueLabel">
       <property name="text">
        <string>Action: 0%</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QPushButton" name="SetInFButton">
       <property name="text">
        <string>Set In F</string>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="QPushButton" name="SetOutFButton">
       <property name="text">
        <string>Set Out F</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="ErrorLabel">
       <property name="text">
        <string>Error: </string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="InValueLabel">
       <property name="text">
        <string>Value: 0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="PointsLabel">
       <property name="text">
        <string>Points: 0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QComboBox" name="PortSelectcomboBox"/>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="ConnectButton">
       <property name="text">
        <string>Connect</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QSpinBox" name="InFspinBox">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>10000</number>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="QSpinBox" name="OutFspinBox">
       <property name="minimum">
        <number>15</number>
       </property>
       <property name="maximum">
        <number>15000</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTabWidget" name="ControlSelectWidget">
    <property name="geometry">
     <rect>
      <x>175</x>
      <y>36</y>
      <width>221</width>
      <height>91</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Manual</string>
     </attribute>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Action</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="ActionspinBox">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>1</y>
        <width>42</width>
        <height>20</height>
       </rect>
      </property>
      <property name="correctionMode">
       <enum>QAbstractSpinBox::CorrectToNearestValue</enum>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="RelayControl">
     <attribute name="title">
      <string>Relay</string>
     </attribute>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>47</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Gate</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="GatespinBox">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>1</y>
        <width>42</width>
        <height>20</height>
       </rect>
      </property>
      <property name="maximum">
       <number>3300</number>
      </property>
      <property name="value">
       <number>10</number>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="PIDControl">
     <attribute name="title">
      <string>PID</string>
     </attribute>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>1</y>
        <width>20</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>P</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>23</y>
        <width>47</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>I</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>42</y>
        <width>47</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>D</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="PIDPdoubleSpinBox">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>1</y>
        <width>42</width>
        <height>20</height>
       </rect>
      </property>
      <property name="decimals">
       <number>1</number>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="PIDIdoubleSpinBox">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>20</y>
        <width>42</width>
        <height>20</height>
       </rect>
      </property>
      <property name="correctionMode">
       <enum>QAbstractSpinBox::CorrectToPreviousValue</enum>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="PIDDdoubleSpinBox">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>40</y>
        <width>42</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="StopControlButton">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>130</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>StopControl</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>175</x>
      <y>10</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TargetValue</string>
    </property>
   </widget>
   <widget class="QPushButton" name="StartUpdateControlButton">
    <property name="geometry">
     <rect>
      <x>175</x>
      <y>130</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>StartControl</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="TargetValuespinBox">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>10</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>3300</number>
    </property>
    <property name="value">
     <number>1000</number>
    </property>
   </widget>
   <widget class="QCustomPlot" name="plotwidget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>501</width>
      <height>191</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="PlotClearButton">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>40</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>ClearPlot</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SaveDateButton">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>10</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Save to file</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="sizeGripEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>524</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <action name="actionSave_to_file">
   <property name="text">
    <string>Save to file</string>
   </property>
  </action>
  <action name="actionClear_Plot">
   <property name="text">
    <string>Clear Plot</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
